app/models/user.rb

```diff
@@ -358,4 +358,8 @@ def record_email_verified_event
   def record_password_update_event
     record_event!(Events::UserEvent::PASSWORD_CHANGED)
   end
+
+  def unique_with_org_handle
+    errors.add(:handle, "has already been taken") if handle && Organization.where("handle = lower(?)", handle.downcase).any?
```

Comment:
```suggestion
    errors.add(:handle, "has already been taken") if handle && Organization.where("lower(handle) = ?", handle.downcase).any?
```

is this what you meant?

Reply:
whoops, yes 🤦🏻